### PR TITLE
documenting the new contributor ambassador role

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Signed-off-by: Arsh Sharma <arshsharma461@gmail.com>

Fixes (partially) #31946

Mentioned the NCAs as the recommended first point of contact for new contributors. Anything else folks feel like we should add here?

cc @divya-mohan0209 @reylejano @natalisucks @jimangel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,6 +167,10 @@ For more information about contributing to the Kubernetes documentation, see:
 - [Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/)
 - [Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/)
 
+### New Contributor Ambassador
+
+If you need help at any point when contributing, the New Contributor Ambassadors are a good point of contact. These are SIG Docs approvers whose responsibilities include mentoring new contributors and helping them through their first few pull requests. The best place to contact the New Contributors Ambassadors would be on the [Kubernetes Slack](https://slack.k8s.io/). 
+
```
Comment (RinkiyaKeDad) [inline README.md:173]: Do you think we should have a table here with the list of NCAs and their Slack and GitHub usernames?

File: README.md [hunk 2]
```diff
@@ -167,6 +167,10 @@ For more information about contributing to the Kubernetes documentation, see:
 - [Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/)
 - [Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/)
 
+### New Contributor Ambassador
```
Comment (sftim) [inline README.md:170]: (nit)
```suggestion
### New contributor ambassadors
```

**Review by sftim — COMMENTED** [review]
If there's any page about this concept in https://k8s.dev/, it's be good to link there.

If not (yet), then: can we help broaden the approach to more SIGs? Perhaps working with SIG ContribEx?

**Comment by sftim** [comment]
Ah
/approve cancel

**Comment by RinkiyaKeDad** [comment]
Updated the PR with the link. @sftim @reylejano do you feel we should document details about new contributor ambassadors on this page, so folks know who to contact? Afaik this isn't documented anywhere currently.

**Comment by divya-mohan0209** [comment]
> Updated the PR with the link. @sftim @reylejano do you feel we should document details about new contributor ambassadors on this page, so folks know who to contact? Afaik this isn't documented anywhere currently.

I think this would be beneficial, tbh.

**Comment by RinkiyaKeDad** [comment]
I've updated the PR with the details @divya-mohan0209, PTAL.

**Pull request merged**